metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/siblings/SiblingAssociationHook.java

```diff
@@ -362,4 +376,67 @@ private Filter createFilterForEntitiesWithYouAsSibling(
     return filter;
   }
 
+  private SubTypes getSubtypesFromEntityClient(
+      final Urn urn
+  ) {
+    try {
+      EntityResponse entityResponse = _entityClient.getV2(
+          DATASET_ENTITY_NAME,
+          urn,
+          ImmutableSet.of(SUB_TYPES_ASPECT_NAME),
+          _systemAuthentication
+      );
+
+      if (entityResponse != null && entityResponse.hasAspects() && entityResponse.getAspects().containsKey(Constants.SUB_TYPES_ASPECT_NAME)) {
+        return new SubTypes(entityResponse.getAspects().get(Constants.SUB_TYPES_ASPECT_NAME).getValue().data());
+      } else {
+        return null;
+      }
+    } catch (Exception e) {
```

Comment:
Is this a specific exception like URISyntax?

Reply:
added specificity